code/modules/admin/view_variables/view_variables.dm

```diff
@@ -8,6 +8,9 @@ var/global/list/view_variables_no_assoc = list("verbs", "contents","screen","ima
 	set category = "Debug"
 	set name = "View Variables"
 
+	debug_variables_inner(D)
+
+/client/proc/debug_variables_inner(datum/D, filter = "" as null|text)
```

Comment:
should probably have a comment explaining why this was done, i believe it's because otherwise using the VV verb would would prompt you to input a filter string

in fact it might be worth pulling the VV changes out into a separate PR so that it can actually be reviewed (by loaf)